airbyte-integrations/connectors/destination-local-json/src/main/resources/spec.json

```diff
@@ -0,0 +1,18 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/destinations/local-json",
+  "supportsIncremental": true,
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Local Json Destination Spec",
+    "type": "object",
+    "required": ["destination_path"],
+    "additionalProperties": false,
+    "properties": {
+      "destination_path": {
+        "description": "Path to the directory where json files will be written. The files will be placed inside that local mount. For more information check out our <a href=\"https://docs.airbyte.io/integrations/destinations/local-json\">docs</a>",
+        "type": "string",
+        "examples": ["/json_data"]
```

Comment:
This path should match what we're doing in local csv. 

I'm fine with just putting it in a relative directory to the mount (and requiring no preceding slash) or doing it how we do it in local-csv where we require it to start with `/local/`). Either way is fine if it matches.

This is how it's defined in local-csv:
```
      "destination_path": {
        "description": "Path to the directory where csv files will be written. Must start with the local mount \"/local\". Any other directory appended on the end will be placed inside that local mount. For more information check out our <a href=\"https://docs.airbyte.io/integrations/destinations/local-csv\">docs</a>",
        "type": "string",
        "examples": ["/local"],
        "pattern": "(^\\/local\\/.*)|(^\\/local$)"
      }
```